includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -385,4 +618,10 @@ private function getLicenseUrl() {
 
 		return $licenseUrl;
 	}
+
+	private function getHref( $hrefKey ) {
+		return isset( $this->hrefs[$this->lang][$hrefKey] )
```

Comment:
It shouldn't, though you can double check to verify. `$foo['bar'] ?? $foo['foo'];` is equivalent to `isset( $foo['bar'] ) ? $foo['bar'] : $foo['foo'];`, it's just syntactic sugar.